### DISCUSSION TITLE
bug: initial load time has more than tripled between the latest releases

### DISCUSSION THREAD

**Discussion by anthonymartin** [open-webui/open-webui#6709] [Issues]
# Bug Report

## Installation Method

The application is deployed via Helm / kubernetes

## Environment

- **Open WebUI Version:** [v0.3.35](https://github.com/open-webui/open-webui/releases/tag/v0.3.35)
- **Ollama (if applicable):** N/A

- **Operating System:** Mac / Android
- **Browser (if applicable):** Chrome/Safari/Firefox

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When loading open webui anytime after having logged in, the page should render with little latency. Version [v0.3.32](https://github.com/open-webui/open-webui/releases/tag/v0.3.32) does not exhibit this latency.

## Actual Behavior:

It takes several seconds to load open webui after having logged in already. 

## Description

**Bug Summary:**
When loading open webui anytime after having logged in, the page should render with little latency. Version [v0.3.32](https://github.com/open-webui/open-webui/releases/tag/v0.3.32) does not exhibit this latency compared to [v0.3.35](https://github.com/open-webui/open-webui/releases/tag/v0.3.35) which does. I am not sure which release between the 2 introduced the latency.

## Reproduction Details

**Steps to Reproduce:**
1. Go to open webui and log in.
2. Reload the page - it will take several second to reload compared to version 0.3.32

**Comment by Tobe2d** [comment]
+1
